prime-dotnet-webapi/Services/EnrolleeService.cs

```diff
@@ -1055,6 +1073,80 @@ public async Task<IEnumerable<HpdidLookup>> HpdidLookupAsync(IEnumerable<string>
                 .ToListAsync();
         }
 
+        public async Task<EnrolleeLookup> GpidLookupAsync(GpidLookupOption option)
```

Comment:
There should be a check that `ConsentForAutoPull` is `true`

Reply:
good catch! it is fixed.